src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10374,14 +10381,16 @@ private TupleExpressionSyntax ParseTupleExpressionTail(SyntaxToken openParen, Ar
                     list.Add(arg);
                 }
 
-                var closeParen = this.EatToken(SyntaxKind.CloseParenToken);
-                var result = _syntaxFactory.TupleExpression(openParen, list, closeParen);
-
                 if (list.Count < 2)
                 {
-                    result = this.AddError(result, ErrorCode.ERR_TupleTooFewElements);
+                    list.AddSeparator(SyntaxFactory.MissingToken(SyntaxKind.CommaToken));
+                    var missingExpression = this.AddError(this.CreateMissingIdentifierName(), ErrorCode.ERR_TupleTooFewElements);
+                    list.Add(_syntaxFactory.Argument(nameColon: null, refOrOutKeyword: default(SyntaxToken), expression: missingExpression));
```

Comment:
>nameColon: null [](start = 53, length = 15)

Is it valid for the second item to have no name when the first one is named? Let's test scenario when the only element is named to make sure no strange errors reported for the element that we are adding here. Same for tuple type.